tests/GroupTest.php

```diff
@@ -445,7 +446,7 @@ public function handle(ServerRequestInterface $request): ResponseInterface
     private function getDispatcher(): MiddlewareDispatcher
     {
         return new MiddlewareDispatcher(
-            new MiddlewareFactory(new Container([])),
+            new MiddlewareFactory(new Container([]), new WrapperFactory(new Container([]))),
```

Comment:
Lots of parentheses - quite hard to read. Maybe refactor using assignments to separate variables

Reply:
Done.